CS483/CS483/Kartaclysm/Components/Abilities/ComponentAbilityIcon.cpp

```diff
@@ -0,0 +1,126 @@
+//----------------------------------------------------------------------------
+// ComponentAbilityIcon.h
+// Author: Bradley Cooper
+//
+// Component that handles the conditions for activating an ability.
+//----------------------------------------------------------------------------
+
+#include "ComponentAbilityIcon.h"
```

Comment:
Nevermind.  Makes sense, since this is completely changing how icons work.  Still, seems like combining multiple things into one commit.